README.template.md

```diff
@@ -59,39 +59,42 @@ There are a number of `docker-compose.yml` files provided by this repository:
 
 ## Override
 
-This repository ignores `docker-compose.override.yml` which will be included in
-any `docker compose` commands you invoke by default.
+This git repository does not track `docker-compose.override.yml` which will 
+be included by default in all `docker compose` commands you invoke.
 
 Two platform dependent templates that allow for access to the hosts `SSH agent`
 are provided for `development` environments.
 
-Simply copy the appropriate `docker-compose.PLATFORM.yml` file into
+Simply copy the appropriate* `docker-compose.PLATFORM.yml` file into
 `docker-compose.override.yml`, on your development machine.
 
 Any additional changes that are for your local / development environment can
-then be added to `docker-compose.override.yml`.
+then be added to `docker-compose.override.yml`. Because they're not tracked...
 
 ## Building
 
-You can build your locally using `docker compose`
+You can build your development environment locally using `docker compose`
 
 ```bash
 docker compose --profile dev build
 ```
+This has the effect of...
```

Comment:
(put here as a placeholder, that this section should at some level explain what building is since you haven't even pulled images yet, I can't imagine what's being built.)